.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ros_distro: [noetic]
```

Comment:
Why not test klited and rolling as well?

Reply:
Sure, we can expand this matrix, but first let's get Humble and Jazzy working. Right now, they're failing differently, which makes me suspect rosbridge

Reply:
ok, after rewriting the test that was failing on both ROS 2 distros (because it was relying on deterministic Topic behavior that isn't present in ROS 2 without EventsExecutor), now Humble is the only one failing. I actually think maybe some code got contributed to roslibjs that is incompatible with Humble, so I'll investigate that.